core/src/main/java/com/epam/pipeline/entity/datastorage/StorageUsage.java

```diff
@@ -31,6 +31,11 @@ public class StorageUsage {
     private String name;
     private DataStorageType type;
     private String path;
+    private Long size;
+    private Long effectiveSize;
+    private Long oldVersionsSize;
+    private Long oldVersionsEffectiveSize;
+    private Long count;
```

Comment:
I think we should either add `effectiveCount` field, or remove `oldVersionsSize`, `oldVersionsEffectiveSize`. Other way it is a little bit inconsistent